server/handlers.go

```diff
@@ -52,80 +57,507 @@ func handleRequestDownloaderMissing(logger *zap.Logger) http.HandlerFunc {
 	}
 }
 
-func handleAccountImages(multiaccountsDB *multiaccounts.Database, logger *zap.Logger) http.HandlerFunc {
-	return func(w http.ResponseWriter, r *http.Request) {
-		params := r.URL.Query()
+type ParsedParams struct {
+	KeyUID          string
+	PublicKey       string
+	ImageName       string
+	ImagePath       string
+	FullName        string
+	InitialsLength  int
+	FontFile        string
+	FontSize        float64
+	Color           color.Color
+	BgSize          int
+	BgColor         color.Color
+	UppercaseRatio  float64
+	Theme           ring.Theme
+	Ring            bool
+	IndicatorSize   int
+	IndicatorBorder int
+	IndicatorColor  color.Color
+
+	AuthorID     string
+	URL          string
+	MessageID    string
+	AttachmentID string
+
+	Hash     string
+	Download bool
+}
 
-		keyUids, ok := params["keyUid"]
-		if !ok || len(keyUids) == 0 {
-			logger.Error("no keyUid")
-			return
+func ParseParams(logger *zap.Logger, params url.Values) ParsedParams {
```

Comment:
Perhaps change this function name to `ParseImageParams`? Just something that describes what is being parsed.